dotnet/src/SemanticKernel.Skills/Skills.Grpc/IGrpcOperationRunner.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Skills.Grpc.Model;
+
+namespace Skills.Grpc;
+
+/// <summary>
+/// Interface for gRPC operation runner classes.
+/// </summary>
+internal interface IGrpcOperationRunner
```

Comment:
In this particular case, at the current phase of development we don't need it. I created it automatically thinking that any code that would need it will depend on it rather than on the class to make that code more flexible (it can use any other implementation if/when needed) and testable (it will be very easy to create a mock interface). So, I'll remove it for now until it's required.